Preprocessing/AudioNormalizer.py

```diff
@@ -0,0 +1,97 @@
+import argparse
+import json
+from pathlib import Path
+import sys
+from typing import Optional
+
+import ffmpeg
+
+from Preprocessing.ErrorLogger import ErrorLogger
+from Preprocessing.utils import setup_logger
+
+
+class AudioNormalizer:
+    SUPPORTED_EXTENSIONS = [".mp4", ".mkv", ".avi"]
+
+    def __init__(self, input_folder: str, output_folder: str):
+        self.input_folder = Path(input_folder)
+        self.output_folder = Path(output_folder)
+        self.logger = setup_logger(self.__class__.__name__)
+        self.error_logger = ErrorLogger(self.logger)
```

Comment:
nie mieliśmy konwencji ustalonej żeby dawać type hinty do static i non-static fieldów klasy? jak tak to do propagacji we wszystkich nowych klaskach